examples/dtls-echo/dtls-server.c

```diff
@@ -122,12 +122,10 @@ static void dtls_handle_read(dtls_context_t *ctx)
 
     if (memcpy(&session.addr, &remote_peer->remote->addr.ipv6, 16) == NULL) {
         puts("ERROR: memcpy failed!");
-        return;
```

Comment:
How can `memcpy` ever fail?
This check seems rather pointless.

Reply:
I try not to touch unrelated code but I'll push a commit to clean up obvious stuff like this.